content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md

```diff
@@ -18,6 +19,13 @@ software:
   - iot-cloud
 ---
 
+
+## Required Hardware and Software
+
+* [Portenta X8](https://store.arduino.cc/products/portenta-x8) board.
+* Wi-Fi AP with Internet access.
+* Arduino Pro Cloud Subscription plan. [Learn more about the Plan](https://www.arduino.cc/pro/hardware/product/portenta-x8).
```

Comment:
```suggestion
* Arduino Pro Cloud Subscription plan. [Learn more about the Plan](https://www.arduino.cc/pro/hardware/product/portenta-x8#pro-cloud).
```

This will scroll the page down to the relevant section in order to provide the reader with quick access to the relevant information.

Reply:
Nice! I couldnt get the anchor right, thanks!!! 🙌